CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Changelog
 * integrate [ohsome-filter](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-filter) module fully into this repository, renaming it to `oshdb-filter` ([#306])
 * rename and move submodules of `oshdb-tool` ([#384])
 * rename some classes, methods and enum constants; move some classes/interfaces ([#369], [#374])
+* move ignite backend implementation into its own submodule ([#387])
```

Comment:
Suggestion: add entry for extracting/moving/creating an `oshdb-api-ignite` package

Reply:
Actually, that's what I meant with this entry. I improved the wording now a bit. Is it clearer now?

Reply:
:+1: